packages/help-center/src/components/help-center-gpt.tsx

```diff
@@ -101,17 +102,19 @@ export function HelpCenterGPT() {
 		delayBetweenWords: 1400,
 	} );
 
-	const doThumbsUp = () => {
+	const doThumbsUp = ( source: string ) => {
 		setFeedbackGiven( true );
 		recordTracksEvent( 'calypso_helpcenter_gpt_response_thumbs_up', {
 			location: 'help-center',
+			answer_source: source,
 		} );
 	};
```

Comment:
If you want to be able to use it as previously as a callback:
```suggestion
	const doThumbsUp = ( source: string ) => {
		return () => {
			setFeedbackGiven( true );
			recordTracksEvent( 'calypso_helpcenter_gpt_response_thumbs_up', {
				location: 'help-center',
				answer_source: source,
			} );
		};
	};
```
Then it could be used as `<Button onClick={ doThumbsUp( data?.source ) }>&#128077;</Button>`

Reply:
This makes so much sense, thanks!